### PR TITLE
Use Prettier to enforce the Google Javascript styleguide, and provide good defaults for markdown and yaml

### PR DESCRIPTION
**Pull request by kmcnellis**
### Description
-Use the google javascript styleguide
-Add a readme badge for the styleguide
-Run on more files (yaml and markdown)

### CODE REVIEW

File: formatter/fmt [hunk 1]
```diff
@@ -37,13 +37,14 @@ else
...
-echo 'Start format TS'
+echo 'Start formatting with prettier'
 npx prettier \
   --write \
   --config /.prettierrc.js \
   --ignore-path /.prettierignore \
-  {server,browser-test}
-echo 'Done formatting TS'
+  --ignore-unknown \
+  .
```
Comment (bion) [inline formatter/fmt:46]: question (blocking): why `.` and not `{server,browser-test}`?

Comment (kmcnellis) [reply formatter/fmt:46]: It's also formatting a bunch of other file formats, including css, yaml, and markdown.

The full list is here: https://prettier.io/docs/en/index.html

**Review by bion — APPROVED** [review]
Please check the output to ensure it isn't attempting to format files generated by SBT or file in node_modules

**Comment by kmcnellis** [comment]
> Please check the output to ensure it isn't attempting to format files generated by SBT or file in node_modules

I don't see any. If you find extra files we don't want formatted, just add them to the .prettierignore

**Pull request merged**